test_gtest/ParenthesesTests.cpp

```diff
@@ -5,12 +5,102 @@
 TEST(ParenthesesTests, givenEmptyStringWhenIsBalancedIsCalledThenResultIsTrue)
 {
     // GIVEN
-    std::string input{};
+    std::string inputEmpty{};
     Parentheses p;
+    // WHEN
+     auto result = p.isBalanced(inputEmpty);
+    // THEN
+     ASSERT_TRUE(result);   
+}
+
+TEST(ParenthesesTests, givenBasicPositivCasesWhenIsBalancedIsCalledThenResultIsTrue)
+{
+    // GIVEN
+    std::vector <std::string> inputBasicPositivCases {"", "()", "{}", "[]"}; 
+    Parentheses p;
+    // WHEN
+     for(auto currentCase : inputBasicPositivCases){
+        auto result = p.isBalanced(currentCase);
+    // THEN
+        ASSERT_TRUE(result);
+     }
+}
+
+TEST(ParenthesesTests, givenBasicPositivCasesWhenIsBalancedIsCalledThenResultIsFalse)
+{
+    // GIVEN
+    std::vector <std::string> inputBasicNegativCases {")(", "}{", "]["}; 
+    Parentheses p;
```

Comment:
Jeśli w testach uzywacie wszędzie tego samego obiektu to można zrobić TEST_F, utworzyć klase ParenthesesTests i wrzucić do niej ten obiekt. Wtedy w każdym teście zaoszczędzicie tę linijkę.